float64 beta1
